boards/common/gd32v/include/board_common.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg <koen@bergzand.net>
+ *               2023 Gunar Schorcht <gunar@schorcht.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_gd32v
+ * @{
+ *
+ * @file
+ * @brief       Common board definitions for GD32VF103 boards
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#ifndef BOARD_COMMON_H
+#define BOARD_COMMON_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "macros/units.h"
+
+/**
+ * @name    Xtimer configuration
+ * @{
+ */
+#define XTIMER_HZ                   MHZ(1)
+#define XTIMER_WIDTH                (16)
```

Comment:
Is this even still used by anything?

Reply:
I was not sure. Since a large number of boards still define it and the `seeedstudio-gd32` board definition used it, I decided to keep it. I can remove to see what happens.

Reply:
Uh sorry, turns out I was wrong about this.
This is still needed.

Reply:
Really, where? I removed it and Murdock was happy and nightlies down't give errors.

Reply:
https://user-images.githubusercontent.com/1301112/215072779-fcb2507d-63ff-4a34-b51d-3d699272fcfe.mp4

This doesn't look like 1s intervals to me :sweat_smile:

Reply:
Which test application is it?

Reply:
`examples/blinky`, see #19205